tests/foreman/api/test_classparameters.py

```diff
@@ -1523,35 +707,6 @@ def test_negative_enable_avoid_duplicates_checkbox(self):
         )
         self.assertEqual(sc_param.read().avoid_duplicates, False)
 
-    @tier1
-    def test_positive_remove_matcher(self):
-        """Removal of matcher from parameter.
-
-        :id: 9018d624-07f2-4fb2-b421-8888c7d324a7
-
-        :steps:
-
-            1. Override the parameter and create a matcher for some attribute.
-            2. Remove the matcher created in step 1.
-
-        :expectedresults: The matcher removed from parameter.
-
-        :CaseImportance: Critical
-        """
-        sc_param = self.sc_params_list.pop()
-        sc_param.override = True
-        sc_param.override_value_order = 'is_virtual'
-        sc_param.update(['override', 'override_value_order'])
-        value = gen_string('alpha')
-        override = entities.OverrideValue(
-            smart_class_parameter=sc_param,
-            match='is_virtual=true',
-            value=value,
-        ).create()
-        self.assertEqual(len(sc_param.read().override_values), 1)
-        override.delete()
-        self.assertEqual(len(sc_param.read().override_values), 0)
-
     @skip_if_bug_open('bugzilla', 1374253)
     @tier1
     def test_positive_impact_parameter_delete_attribute(self):
```

Comment:
Looks like you are using more than 2-3 entities could be an ideal case for tier2?